R/pkg/R/column.R

```diff
@@ -44,6 +44,9 @@ setMethod("initialize", "Column", function(.Object, jc) {
   .Object
 })
 
+#' @rdname column
+#' @name column
+#' @aliases column,jobj-method
```

Comment:
Yeah thats an interesting point - The reason this comes up is that we export the `column(character)` function in https://github.com/apache/spark/blob/b4baf086ca380a46d953f2710184ad9eee3a045e/R/pkg/R/functions.R#L273 and if we export a generic then all implementations of that generic automatically seem to get exported. This is the same issue as [SPARK-16519](https://issues.apache.org/jira/browse/SPARK-16519)